docs/comm_protocol.md

```diff
@@ -7,19 +7,22 @@ For implementation on the STM32 side, please refer to the example [here](https:/
 
 For implementation on the Jetson side, please refer to the main communicator [here](../Communication/communicator.py).
 
-# Protocol from Jetson to STM32
+# Protocol between Jetson and STM32
```

Comment:
Right now the protocol does not distinguish packets to/from STM32. Has the protocol been unified so that the same packet structs are used on both ends?

Reply:
Saying they are the "same" structs might not be accurate. We can still define one-sided structs and they will work, depending on how we change the Pack and Unpack functions accordingly.

But yes, now all packet structs are unified on both ends.